py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -31,14 +31,37 @@ def aten_ops_batch_norm(
         target,
         SourceIR.ATEN,
         name,
-        args[0],
-        args[1],
-        args[2],
-        args[3],
-        args[4],
-        args[5],
-        args[6],
-        args[7],
+        input=args[0],
```

Comment:
Consider adding support/decorator for [`native_batch_norm`](https://github.com/pytorch/pytorch/blob/1df14f1bf8661f14e3ebaaf441844ab0cd8534b3/aten/src/ATen/native/native_functions.yaml#L4162) as well.